cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
```

Comment:
- Rename The Class Name With Capital Character for the first Character to be like this "PaymentViewController".